ktor-client/ktor-client-core/ktor-client-core-curl/src/io/ktor/client/call/TypeInfoCurl.kt

```diff
@@ -0,0 +1,14 @@
+package io.ktor.client.call
+
+
+actual interface Type {}
```

Comment:
Please consider reusing the `core-ios` module. Let's make it `core-native` and use external `NativeEngine` with iOS and curl